CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(CMAKE_DEBUG_POSTFIX "d" CACHE STRING "Override CMAKE_DEBUG_POSTFIX.")
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/builds/cmake/Modules)
 
 include(ConfigureWindows)
+include(GNUInstallDirs)
```

Comment:
Thats not needed, we already have this include. (pls git commit amend and force push to fix this).

Reply:
I'll test again but it wasn't working until I included it explicitly.

Reply:
That's because the existing include is after the .pc config. I'll amend to move it up.

Reply:
Rebased.